README.md

```diff
@@ -43,22 +43,22 @@ hub credibility.
 
 These and prior disagreements have now made clear the need for an alternative
 hub with a renewed focus and Alignment to serve as the canonical minimal
-IBC/ICS token hub with respect to Cosmos to champion the ideals of sovereignty,
+IBC/ICS token hub for Cosmos to champion the ideals of sovereignty,
 security, and decentralization everywhere; and secondarily to serve as the main
 base for a political party and more-intelligent voting bloc with respect to
-Gaia to save Gaia from itself. A modification to the distribution of $ATONE
-through slashing those who voted in favor of #848 would help ensure that the
+Gaia to save Gaia from itself. A modification to the distribution of $ATONE,
+a through slashing of those who voted in favor of #848 would help ensure that the
 resultant distribution is more intelligent about security and would make us
 anti-fragile against even the most powerful of adversaries.
 
 ----------------------------------------
 # Vision and Missions
 
 The vision behind this AtomOne fork is to be an alternative minimal fork of
-Gaia ("cosmoshub4") running alongside Gaia to prepare for all contingencies,
+Gaia ("cosmoshub4") running alongside Gaia to prepare for all contingencies
```

Comment:
ok, because "to operate as a political party" is a dependent clause? 
https://www.grammarly.com/blog/comma-before-and/

Reply:
Yes.